static_files/assets/scss/components/_typeahead.scss

```
@@ -1,3 +1,71 @@
 .typeahead.dropdown-menu {
   opacity: 1 !important; // sass-lint:disable-line no-important
 }
+
+.twitter-typeahead {
+  width: 100%;
+}
+
+.form-control {
+  width: 100%;
+}
+
+.tt-menu {
+  background-color: #000;
```

we can use athena variables here for the color:
```suggestion
  background-color: $black;
```